.github/CONTRIBUTING.md

```diff
@@ -99,6 +99,53 @@ To create revisions, you can use [`alembic.sh`](bin/alembic.sh) in the `bin` dir
+| `MYBOT_TOKEN`         | Required | [Create a bot](https://discord.com/developers/applications) and copy the token |
+| `TOPGG_AUTH`          | Optional | Used to sync top.gg                                                            |
+| `TOPGG_TOKEN`         | Optional | Used to sync top.gg                                                            |
+| `TRANSLATOR_SERVICES` | Optional | Will be deprecated                                                             |
+| `MS_TRANSLATE_KEY`    | Optional | Required if "microsoft" is set in `TRANSLATOR_SERVICES`                          |
+| `MS_TRANSLATE_REGION` | Optional | Required if "microsoft" is set in `TRANSLATOR_SERVICES`                          |
+
+
+Then, create a `config.toml` ([TOML](https://toml.io/en/)) with the following values:
+| Key              | Requirement | Description                                               |
+|------------------|-------------|-----------------------------------------------------------|
+| `SUPPORT_GUILD_ID` | Required    | The bot need to be member and administrator of this guild |
+| `BOT_ID`           | Required    | Used for links and top.gg (if enabled)                    |
+| `BOT_NAME`         | Required    | Used for the webhook logs                                 |
+| `LOG_WEBHOOK_URL`  | Optional    | Used to send bot logs using a webhook                     |
+| `OWNER_IDS`        | Required    | Grant permissions such a eval, extensions reload...       |
+
+## Extra informations
+
+In the project structure, `main.py` serves as the entry point executed by Docker. It provides a compact CLI utility with various options that can be utilized through pre-created shell files in the `bin/` directory.
+`mybot.py` is the base of MyBot, with the `MyBot` class, instantiated once at launch, and available in many places in the code.
+
+The `MyBot` class has some utility functions like `getch_user`, `getch_channel`, or `get_or_create_db`. Refer to their docstring for more information.
+
+The `core` directory container internally utilized code for MyBot, while `cogs` contains the implementation of features exposed by MyBot. Additionally, `libraries` holds wrappers for external APIs and tools utilized within the project.
+
+### i18n
+
+All strings should be inside the `_` function from `core.i18n` to have them translated (e.g., `_("Hello World")`). This functions also supports format options like the `str.format()` function (e.g., `_("Hello {}", name)`).
+
+`_` is used to extract the strings from the code automatically, but is also serves as a ✨ magic ✨ function because it walks through the callstack to find an `Interaction` object.  
+Consequently, if you use `_` outside a command callback, it will not be able to retrieve the language. You can then specify it with the `_locale` argument. Set `_locale` to `None` if the string should not be translated at this time in the execution.
```

Comment:
Be more direct in your formulations: the first sentence can be simplified into

```markdown
Using `_` outside a command callback will not retrieve the language.
```

This way of writing is more easier to understand by readers (especially in long and rich technical documents like this one), and is generally more "natural" to English readers, whereas your way of writing is really French-y 🥖